ballerina/http_types.bal

```diff
@@ -137,6 +137,20 @@ public type HeaderValue record {|
     map<string> params;
 |};
 
+# Represents a Server Sent Event emitted from a service.
+public type SseEvent record {|
+    # Name of the event
+    string event?; 
+    # Id of the event
+    string id?;
+    # Data part of the event
+    string data?; 
+    # The reconnect time on failure in milliseconds.
+    int 'retry?;
+    # Comment added to the event
+    string comment?;
+|};
+
 # Dummy types used in the compiler plugin
-type ResourceReturnType Response|StatusCodeResponse|anydata|error;
+type ResourceReturnType Response|StatusCodeResponse|anydata|stream<SseEvent, error?>|error;
```

Comment:
We need to change here as well to fix the diagnostic message for invalid return types: https://github.com/ballerina-platform/module-ballerina-http/blob/ee33f3f21a636d865352def65fdd1463b247e7fb/compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/Constants.java#L87

Reply:
Fixed with: [Address review suggestions](https://github.com/ballerina-platform/module-ballerina-http/pull/2088/commits/0dbfd308603255a46181bbfde7d0edb45fcb8590)